selfdrive/controls/lib/pathplanner.py

```diff
@@ -3,7 +3,7 @@
 from common.numpy_fast import interp
 from selfdrive.controls.lib.latcontrol_helpers import model_polyfit, calc_desired_path, compute_path_pinv
 
-CAMERA_OFFSET = 0.06  # m from center car to camera
+CAMERA_OFFSET = 0.08  # m from center car to camera
```

Comment:
I also can't merge this.
Your camera location or car specific modifications need to be kept local, as these charges negatively affect other users.